platforms/core-runtime/logging/src/main/java/org/gradle/internal/logging/console/DefaultUserInputReceiver.java

```diff
@@ -16,18 +16,52 @@
 
 package org.gradle.internal.logging.console;
 
+import com.google.common.base.CharMatcher;
+import org.apache.commons.lang.StringUtils;
+import org.gradle.internal.Either;
+import org.gradle.internal.logging.events.OutputEventListener;
+import org.gradle.internal.logging.events.PromptOutputEvent;
+
+import javax.annotation.Nullable;
 import java.util.concurrent.atomic.AtomicReference;
 
 public class DefaultUserInputReceiver implements GlobalUserInputReceiver {
     private final AtomicReference<UserInputReceiver> delegate = new AtomicReference<UserInputReceiver>();
+    private OutputEventListener console;
+
+    public DefaultUserInputReceiver() {
+    }
```

Comment:
💅 Unnecessary empty constructor.

Reply:
Don't know what happened there. It's gone.